main.py

```diff
@@ -1,7 +1,31 @@
+from db.models import Actor, Genre
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
 
 
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Dramma"]
+    actors = [
+        ("George", "Klooney"),
+        ("Kianu", "Reaves"),
+        ("Scarlett", "Keegan"),
+        ("Will", "Smith"),
+        ("Jaden", "Smith"),
+        ("Scarlett", "Johansson")
+    ]
+    Genre.objects.bulk_create([Genre(name=genre) for genre in genres])
+    Actor.objects.bulk_create([
```

Comment:
You can skip creating the intermediate lists and directly pass the generator expressions to the bulk_create() method